tests/test_configuration.py

```diff
@@ -444,3 +465,4 @@ def test_backup_for_traces_endpoint(self):
             configuration._ATTRIBUTE_HOST_NAME = (
                 original_attribute_host_name
             )
+            set_tracer_provider(TracerProvider())
```

Comment:
yes, if not when we hit [this](https://github.com/lightstep/otel-launcher-python/pull/108/files#diff-8e9ebccae9617481b5f09ccb18175e61f9871dadd261d0636abb5f6c48f53502L74) line the test will fail because `None` object does not have a `shutdown` attribute.

Reply:
Hmmm.... so the teardown is dependent on this test to complete successfully? this seems strange...

Reply:
Fixed, there is no need to call `shutdown` on the tracer provider since we are resetting initial values in `setUp`.